java/jdbc-driver-graalvm-nativeimage/README.md

```diff
@@ -0,0 +1,2 @@
+# jdbc-driver-graalvm-nativeimage
+[Introduction to Oracle JDBC Driver Configuration for GraalVM Native Image](https://juarezjunior.medium.com/introduction-to-oracle-jdbc-driver-configuration-for-graalvm-native-image-6130e699d03b)
```

Comment:
The README should include enough information to understand the purpose of this code sample and how to use it.

Reply:
As we talked during the first call, all README files under oracle-db-examples/java will be reviewed (new ones like this) and existing ones (old in the repo) to add richer guidance with markdown as part of another initiative as agreed during our call with Santanu